Sources/TensorFlow/Initializers.swift

```diff
@@ -548,3 +548,24 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
         self = Tensor.glorot(fromStandardNormalScaledBy: normal, shape: shape)
     }
 }
+
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    init(
+        orthogonal shape: TensorShape,
+        gain: Scalar = 1,
+        seed: (Int32, Int32) = (Int32.random(in: Int32.min..<Int32.max),
```

Comment:
For usability, we've been making the Tensor API take `Int`'s wherever possible. (Note: this will require you to do the appropriate casting within the initializer in Swift between the variable- and fixed-width number formats.)

Reply:
This is actually not what we are doing for random seeds. We only use `Int` for shape dimensions and count-like numbers. Random seeds are backend specific (in this case, TensorFlow graph specific). Other initializers are also using `Int32` seeds. Usability problems are small for random seeds because they are usually literals and are not part of complicated data flow.

Reply:
based on @rxwei comment I left this unchanged.